<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>486</width>
    <height>476</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>486</width>
    <height>476</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>479</width>
       <height>319</height>
      </rect>
     </property>
     <widget class="QTableWidget" name="films_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>321</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Драма</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Мелодрама</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Документальный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Анимация</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Биография</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Боевик</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ужасы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Комедия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Приключения</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Детектив</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Музыка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Мистика</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Фантастика</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Боевик</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Война</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Семейный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Триллер</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Фэнтези</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вестерн</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Короткометражный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Исторический</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Нуар</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="line">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_begin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>6900</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_end">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>6900</number>
    </property>
    <property name="value">
     <number>6900</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Длительность</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>От</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>До</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Жанр</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Год</string>
    </property>
   </widget>
   <widget class="QComboBox" name="yearBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>В</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ДО</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ПОСЛЕ</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="year">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
